baiji/config.py

```diff
@@ -76,6 +94,13 @@ def key(self):
     def secret(self):
         return self._try(['AWS_SECRET_ACCESS_KEY', 'AWS_SECRET'], 'AWS_SECRET')
 
+    @property
+    def region(self):
+        return self._try(['AWS_DEFAULT_REGION'], 'REGION')
+
+# Alias to avoid breaking changes
+Credentials = Settings
```

Comment:
Right I think it's simpler. Reorganized as you suggested, could you take a look?